packages/sdk-codegen/src/sdkModels.ts

```diff
@@ -1836,24 +1841,40 @@ export class EnumType extends Type implements IEnumType {
       )
     }
     if (methodName) {
-      this.description = `Type defined in ${methodName}\n${this.description}`
+      this.description = `${this.description}${
+        this.description ? ' ' : ''
+      }(Enum defined in ${methodName})`
```

Comment:
I'm curious why the python generated code doesn't get this description addition?

Reply:
it does. See the updated python.gen.spec.ts test

Reply:
guess I missed that because I also didn't see any changes in python generated models (where-as I do for TS generated models) - I feel like when we make a generator change like this we should also updated sdks so that the next person to do it isn't surprised